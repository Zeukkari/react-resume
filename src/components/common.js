import React from 'react'

import { Box, Heading } from 'grommet'

export const ResumeSection = ({ children }) => (
  <Box
    border={{
      side: 'all',
      color: 'red',
      size: 'xsmall',
      style: 'dashed',
    }}
    margin="none"
    pad="none"
  >
    {children}
  </Box>
)

export const SectionTitle = ({ children }) => (
  <Heading level={1} size="medium" align="left">
    {children}
  </Heading>
)
