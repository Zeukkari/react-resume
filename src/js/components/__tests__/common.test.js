import React from 'react'
import renderer from 'react-test-renderer'

import { ResumeSection, SectionTitle } from '../common'

test('renders common resume section', () => {
  const component = renderer.create(<ResumeSection />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

test('renders common resume title', () => {
  const component = renderer.create(<SectionTitle />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
