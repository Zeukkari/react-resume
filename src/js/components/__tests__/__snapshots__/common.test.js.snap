// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders common resume section 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 lhNMsi"
/>
`;

exports[`renders common resume title 1`] = `
<h1
  className="StyledHeading-sc-1rdh4aw-0 jVpYfH"
  size="medium"
/>
`;
